addons/takistan/config.cpp

```diff
@@ -0,0 +1,90 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+            "TACU_Takistan_U_TNA_Major", // TNA
+            "TACU_Takistan_U_TNA_Lieutenant",
+            "TACU_Takistan_U_TNA_NCO",
+            "TACU_Takistan_U_TNA_Rifleman",
+            "TACU_Takistan_U_TNA_Rifleman_02",
+            "TACU_Takistan_U_TNA_Rifleman_03",
+            "TACU_Takistan_U_TNA_Autorifleman",
+            "TACU_Takistan_U_TNA_Pilot",
+            "TACU_Takistan_U_TNA_Commandos_NCO",
+            "TACU_Takistan_U_TNA_Commandos_Breacher",
+            "TACU_Takistan_U_TNA_Commandos_Grenadier",
+            "TACU_Takistan_U_TNA_Commandos_Autorifleman",
+            "TACU_Takistan_U_TNA_Commandos_Medic",
+            "TACU_Takistan_U_TNA_Commandos_Rifleman",
+            "TACU_Takistan_U_TNP_Officer", // TNP
+            "TACU_Takistan_U_TNP_Rifleman",
+            "TACU_Takistan_U_TNP_Rifleman_02",
+            "TACU_Takistan_U_TNP_Autorifleman",
+            "TACU_Takistan_U_TSF_Grenadier", // TSF
+            "TACU_Takistan_U_TSF_Rifleman_01",
+            "TACU_Takistan_U_TSF_Rifleman_02",
+            "TACU_Takistan_U_TSF_Rifleman_03",
+            "TACU_Takistan_U_TSF_Sniper",
+            "TACU_Takistan_U_Tehrik_Leader", // Tehrik
+            "TACU_Takistan_U_Tehrik_Rifleman",
+            "TACU_Takistan_U_Tehrik_Rifleman_02",
+            "TACU_Takistan_U_Tehrik_Grenadier",
+            "TACU_Takistan_U_Tehrik_Autorifleman",
+            "TACU_Takistan_U_Tehrik_Rifleman_AT",
+            "TACU_Takistan_U_Tehrik_Enlisted_Squadleader",
+            "TACU_Takistan_U_Tehrik_Enlisted_Rifleman",
+            "TACU_Takistan_U_Tehrik_Enlisted_Rifleman_02",
+            "TACU_Takistan_U_Tehrik_Enlisted_Rifleman_03",
+            "TACU_Takistan_U_Tehrik_Enlisted_Rifleman_04",
+            "TACU_Takistan_U_Tehrik_Enlisted_Autorifleman",
+            "TACU_Takistan_U_Tehrik_Enlisted_Rifleman_AT",
+            "TACU_Takistan_V_TNA_HMMWV_M2", // Vehicles - TNA
+            "TACU_Takistan_V_TNA_M1151_M2",
+            "TACU_Takistan_V_TNA_M1151_M2_Deploy",
+            "TACU_Takistan_V_TNA_HMMWV_Unarmed",
+            "TACU_Takistan_V_TNA_HMMWV_Transport",
+            "TACU_Takistan_V_TNA_Kamaz",
+            "TACU_Takistan_V_TNA_Kamaz_Open",
+            "TACU_Takistan_V_TNA_Kamaz_Ammo",
+            "TACU_Takistan_V_TNA_Kamaz_Fuel",
+            "TACU_Takistan_V_TNA_M113_Med",
+            "TACU_Takistan_V_TNA_Mi17",
```

Comment:
```suggestion
            "TACU_Takistan_V_TNA_Mi17",
            "TACU_Takistan_V_TNA_M2_Static",
            "TACU_Takistan_V_TNA_M2_Mini_Static",
```
(Turrets are also vehicles)